src/pages/quiz/Quiz.tsx

```diff
@@ -41,11 +54,25 @@ const QuizPage = () => {
   const handleNonInputChange = (key: string) => (value: string | string[]) =>
     handleNonInputValueChange(key, value)
 
-  const { loading, response } = useAxios<Quiz, string>({
-    service: getQuiz
+  const {
+    data: response,
+    isLoading,
+    isError
+  } = useQuery<Quiz, string>({
```

Comment:
You don't need to type `useQuery`, it's already typed by service
```suggestion
  } = useQuery({
```